### DISCUSSION TITLE
How to properly add options to Flows?

### DISCUSSION THREAD

**Discussion by nullndr** [taskforcesh/bullmq#1166] [Q&A]
I have two little jobs A and B that have a parent child relationship

```typescript
  await flowProducer.add(
    {
      name: "do something with A",
      queueName: queueA.name,
      data: job_A_data,
      children: [
        {
          name: "do something with B",
          queueName: queueB.name,
          data: job_B_data,
        },
      ],
    },
    {
      queuesOptions: {
        [queueA.name]: {
          defaultJobOptions: {
            repeat: {
              every: 60000, // every 60 seconds
            },
          },
        },
      },
    }
  );
```

This approach does not works, `jobA` is executed only once.

Do I need to add also something for `jobB`?

This [example](https://docs.bullmq.io/guide/flows#provide-options) can't help me because the example use the same queue, but I need to use two different queues.

**Comment by nullndr** [comment]
I looked at this [issue](https://github.com/taskforcesh/bullmq/issues/851).

Are repeatable flows still not supported?

If not, why `FlowQueuesOpts` still allow it?

**Reply by roggervalf** [reply]
hi @nullndr, sorry for the late response and thank you for reporting it, it sounds reasonable to remove repeat option also in FlowQueuesOpts.

**Reply by roggervalf** [reply]
hey, I double check our api and https://api.docs.bullmq.io/interfaces/DefaultJobOptions.html we currently don't have repeat property into https://api.docs.bullmq.io/interfaces/DefaultJobOptions.html, which version are you using?

**Reply by nullndr** [reply]
@roggervalf this was one year ago, sorry but I don't recall the version I was using

**Reply by roggervalf** [reply]
no problem, just to clarify, this should be solved in last versions
